src/import/sb2import.js

```diff
@@ -90,8 +113,8 @@ function parseScratchObject (object, runtime, topLevel) {
     target.isStage = topLevel;
     // The stage will have child objects; recursively process them.
     if (object.children) {
-        for (var j = 0; j < object.children.length; j++) {
-            parseScratchObject(object.children[j], runtime, false);
+        for (var m = 0; m < object.children.length; m++) {
+            parseScratchObject(object.children[m], runtime, false);
```

Comment:
@Jonathan50   Browser support - Safari 9, IE < 11, Android Browser 4.4, etc. all don't support `let`. But we hope to in the future...